src/ca65/pseudo.c

```diff
@@ -421,7 +421,133 @@ static void DoAlign (void)
     SegAlign (Alignment, (int) FillVal);
 }
 
+static void ConvertEscapesStrBuf (StrBuf *Data)
+{
+  char *bufin;
+  char *bufout;
+  size_t len;
+  size_t byte_len;
+  char *s;
+
+  len = SB_GetLen(Data);
+  
+  if((bufin = (char *) malloc(len+1)) == NULL)
+    {
+      perror("malloc");
+      exit(1);
+    }
+  
+  strcpy(bufin, SB_GetBuf(Data));
```

Comment:
Changed advice:  Use `xstrdup (SB_GetConstBuf (Data))`.  It will do the `malloc()` and the `strcpy()`.